### PR TITLE
Chore: add postgres example

### PR DESCRIPTION
**Pull request by sayjeyhi**

### CODE REVIEW

File: examples/08_postgres/src/Users.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// @ts-ignore
+import db from "./db.js";
+
+export const UsersList = async ({ searchText }: { searchText: string }) => {
+  // WARNING: This is for demo purposes only.
+  // We don't encourage this in real apps. There are far safer ways to access
+  // data in a real application!
+  const users = (await db.query(
```
Comment (dai-shi) [inline examples/08_postgres/src/Users.tsx:8]: UserList.tsx imported from Couter.tsx, which is a client component.
In that case, you can't use `db` here under client component tree.
You want to import UserList.tsx from App.tsx, which is a server component.

Comment (sayjeyhi) [reply examples/08_postgres/src/Users.tsx:8]: Is it a part of the limitations of `wakuwork` implementation or it is a global concept?
cuz I saw documents like this, which allows importing the Server component into the Client component.
4cisrfhurwmv7ja9b4ei

Comment (dai-shi) [reply examples/08_postgres/src/Users.tsx:8]: It's not the limitation of wakuwork. It's how RSC is designed.
The shared component can't have server specific code.

**Comment by dai-shi** [comment]
ref: https://github.com/dai-shi/wakuwork/issues/17#issuecomment-1535566006

Closing this for now, but I'm _very_ curious how people use it.

**Pull request closed without merging**